File: Person.py
class Person:
    def __init__(self, newName="none", address="none", phone="999-999-9999"):
        self.name = newName
        self.address = address
        self.phone = phone

    def setName(self, name):
        self.name = name
    
    def getName(self):
        return str(self.name)

    def setAddress(self, address):
        self.address = address

    def getAddress(self):
        return str(self.address)

    def setPhone(self, phone):
        self.phone = phone

    def getPhone(self):
        return str(self.phone)

    def __str__(self):
        breaking = '==================='
        string = f'Name: {self.getName()}\nAddress: {self.getAddress()}\nPhone: {self.getPhone()}\n{breaking}'
        return string

File: Person.java
public class Person{
    public String name;
    public String address;
    public String phone;
    public Person(String newName, String address, String phone){
        this.name = newName;
        this.address = address;
        this.phone = phone;
    }
    public void setName(String name){
        this.name = name;
    }
    public String getName(){
        return this.name;
    }
    public void setAddress(String address){
        this.address = address;
    }
    public String getAddress(){
        return this.address;
    }
    public void setPhone(String phone){
        this.phone = phone;
    }
    public String getPhone(){
        return this.phone;
    }
    public String toString(){
        return "Name: " + getName() + ", Address: " + getAddress() + ", Phone: " + getPhone();
    }
}